src/core/Akka.Tests/Actor/ActorLifeCycleSpec.cs

```diff
@@ -393,7 +393,12 @@ public async Task Call_PreStart_with_correct_message_and_sender()
                 c.OnPreRestart = (ex, mess, context) =>
                 {
                     TestActor.Tell(ex);
-                    TestActor.Tell(mess);
+                    
+                    // can't relay the Restart back because that will blow up the TestActor
+                    if (mess is not IntentionalRestart)
```

Comment:
Can't reply back with the `IntentionalRestart` message because it will crash the `TestActor`.